### PR TITLE
cpSpacePointQueryFirst returns Shape object when it should return null

### PR DESCRIPTION
**Pull request by smshuja**
When calling space.pointQueryFirst() through JSB, it always returns a shape even if no shape is found and calling any function on this value results in Bad Access error. I have added a check to return null if no shape is found.

### CODE REVIEW

**Comment by smshuja** [comment]
Any idea why Travis CI build failed? I checked the build log but doesn't seem to be anything related to my commits.

File: samples/Javascript/Shared [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit bfa9252643fe4463dafdec5cf491cc1c9c406a12
+Subproject commit 999fabb306cddffd0c043ee1cff6e8627ac0df24
```
Comment (dumganhar) [inline samples/Javascript/Shared:1]: Why to change the submodule reference?

File: scripting/javascript/bindings/generated [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 07c06b1537159e3f9828039a14ea0a80df05ec7c
+Subproject commit 2f59a8dbc553ea2ef966dc595684f027a6d56931
```
Comment (dumganhar) [inline scripting/javascript/bindings/generated:1]: Why to change the submodule reference?

**Comment by dumganhar** [comment]
Failed — The Travis CI build failed (Details), you could find what the compilation error here(https://travis-ci.org/cocos2d/cocos2d-x/jobs/7213436)

**Comment by dumganhar** [comment]
Your changes seem great, but please revert the modification of the submodule reference.

File: samples/Javascript/Shared (hunk 1 above)
Comment (smshuja) [inline samples/Javascript/Shared:1]: Sorry for that, I am new to git and github, it happened when I pulled the changes from the cocos2d-x master

**Comment by smshuja** [comment]
I guess I will have to create a branch and send a pull request.

**Pull request closed without merging**